test/Test/HTTPure/HeadersSpec.purs

```diff
@@ -2,11 +2,13 @@ module Test.HTTPure.HeadersSpec where
 
 import Prelude
 
+import Data.Array.NonEmpty (fromArray, singleton)
 import Data.Maybe (Maybe(Nothing, Just))
 import Data.Tuple (Tuple(Tuple))
 import Effect.Class (liftEffect)
-import HTTPure.Headers (empty, header, headers, read, write)
 import HTTPure.Lookup ((!!))
+import HTTPure.RequestHeaders (empty, read)
+import HTTPure.ResponseHeaders (header, header', headers, write)
```

Comment:
With `HTTPure.Headers` split into two modules, I think we should split this test into `Test.HTTPure.RequestHeadersSpec` and `Test.HTTPure.ResponseHeadersSpec`.